app/dashboard/models.py

```diff
@@ -1554,6 +1554,7 @@ class Activity(SuperModel):
         ('update_grant', 'Updated Grant'),
         ('killed_grant', 'Cancelled Grant'),
         ('new_grant_contribution', 'Contributed to Grant'),
+        ('new_grant_subscription', 'Subscribed to Grant'),
```

Comment:
killed_grant_contribution?
new_grant?

Reply:
If we want to differentiate one-off contributions from recurring subscriptions, we need this new activity type. I assumed that's what you wanted when you said

> grant activity feed items for 'new subscription', 'new contribution', and 'cancel subscription' on gitcoin profile

Reply:
i dont necessarily want to differentiate those things.  i'm just pointing out that you haven't added activity types for killed grant contributions or for new grants.

Reply:
because they already exist

Reply:
derp derp